packages/wallets/src/features/cashier/modules/Transfer/components/TransferFormAmountInput/TransferFormAmountInput.tsx

```diff
@@ -103,14 +92,18 @@ const TransferFormAmountInput: React.FC<TProps> = ({ fieldName }) => {
     );
 
     const onTimerCompleteHandler = useCallback(() => {
-        if (!toAccount?.currency || !exchangeRate?.rates) return;
-
-        const toRate = exchangeRate.rates[toAccount.currency];
-        const convertedToAmount = Number((fromAmount * toRate).toFixed(toAccount?.currencyConfig?.fractional_digits));
-        setFieldValue('toAmount', convertedToAmount);
+        refetchExchangeRatesAndLimits().then(res => {
```

Comment:
v.minor / preference: what about async/await ?

Reply:
I personally prefer `then`-ing but can rewrite to `async/await` if that's more preferred as a pattern, right now I believe we don't have a guideline on what's a better approach